src/main/java/step4_ladderGame/domain/LadderGame.java

```diff
@@ -0,0 +1,32 @@
+package step4_ladderGame.domain;
+
+
+import step4_ladderGame.dto.LadderDto;
+import step4_ladderGame.dto.PlayersDto;
+
+public class LadderGame {
+
+    private final Ladder ladder;
+    private final Players players;
+
+    private LadderGame(Ladder ladder, Players players) {
+        this.ladder = ladder;
+        this.players = players;
+    }
+
+    public static LadderGame of(String[] names, int height) {
+        return new LadderGame(Ladder.of(height, names.length, RandomPointStrategy.getInstance()), Players.of(names));
```

Comment:
```suggestion
        return new LadderGame(
            Ladder.of(height, names.length, RandomPointStrategy.getInstance()),
             Players.of(names)
         );
```

이런식으로 줄바꿈을 해주면 더 보기 좋을 것 같아요!